salt/modules/boto_dynamodb.py

```diff
@@ -233,7 +265,7 @@ def describe(table_name, region=None, key=None, keyid=None, profile=None):
     return table.describe()
 
 
-def _extract_index(index_data, global_index=False):
+def extract_index(index_data, global_index=False):
```

Comment:
Another option would be to exempt it from this check by adding the function to the list present here: https://github.com/saltstack/salt/blob/develop/tests/integration/modules/sysmod.py#L29